code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -519,3 +519,101 @@
 	new /obj/item/clothing/glasses/meson(src)
 	new /obj/item/card/id/golem(src)
 	new /obj/item/flashlight/lantern(src)
+
+#define RECURSION_PANIC_AMOUNT 10
+#define LEGACY_TC 5 // How much TC our current value is compared to 2022. Update this any time the value of TC is inflated
```

Comment:
This is also sort of unclear -- this is just the point at which they'll be given raw TC back, right? I feel like `LEGACY_TC` could have a better name.